ocpmodels/common/utils.py

```diff
@@ -211,7 +211,7 @@ def setup_imports():
     datasets_folder = os.path.join(root_folder, "datasets")
     datasets_pattern = os.path.join(datasets_folder, "**", "*.py")
```

Comment:
Following up with your below comment, did you have to also remove the "**" in datasets_pattern?z
(since you mention that the error is thrown for any subdir for datasets as well)

Reply:
No because there are no subdirectories in `datasets` and `trainers`.